testsupport/tiers/checks.go

```diff
@@ -883,7 +883,7 @@ func clusterResourceQuotaDeployments() clusterObjectsCheckCreator {
 			require.NoError(t, err)
 			hard[count("deploymentconfigs.apps")], err = resource.ParseQuantity("30")
 			require.NoError(t, err)
-			hard[count(corev1.ResourcePods)], err = resource.ParseQuantity("50")
+			hard[count(corev1.ResourcePods)], err = resource.ParseQuantity("150")
```

Comment:
This won't work... `clusterResourceQuotaDeployments()` is used for testing all the tiers. We have it because until now all tiers used the same pod quota. But now we are changing the appstudio tier quota only. So you will have to modify line 476:
https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/tiers/checks.go#L476
And probably have a parameterized version of  `clusterResourceQuotaDeployments()` to be used in line 476.